src/stores/constant.store.ts

```diff
@@ -52,5 +54,6 @@ export class ConstantStore {
         this.DB_CONNECTION_STRING = process.env.DB_CONNECTION_STRING || "";
         this.DB_NAME = process.env.DB_NAME || "";
         this.INITIAL_PEER = process.env.INITIAL_PEER || "";
+        this.DEFAULT_STAKE_AMOUNT = 1;
```

Comment:
There already exists a constant for this, it's name is rather odd though; `GENESIS_STAKE_AMOUNT`. We should probably change its name to `default` and remove your newly added const.